media/js/base/mozilla-client.js

```diff
@@ -123,17 +185,6 @@ if (typeof Mozilla === 'undefined') {
             };
 
             callback(details);
-
-            // Log the info with GA. Remove this once we have sufficient data
-            window.dataLayer = window.dataLayer || [];
-            window.dataLayer.push({
```

Comment:
It may take a little while for Gareth to get us GA data from the last PR, so to avoid blocking the merge of this change, should we leave this in for now? We can always file a follow up bug once we have enough data.